site/content/en/docs/Installation/_index.md

```diff
@@ -31,15 +31,30 @@ AKS, while also ensuring that all Cloud providers can support that version.
 
 The following container operating systems and architectures can be utilised with Agones:
 
-
 | OS        | Architecture | Support    |
-|-----------|--------------|------------|
+| --------- | ------------ | ---------- |
 | linux     | `amd64`      | **Stable** |
 | linux     | `arm64`      | Alpha      |
 | [windows] | `amd64`      | Alpha      |
 
 For all the platforms in Alpha, we would appreciate testing and bug reports on any issue found.
 
+## Agones and Kubernetes Supported Versions
+
+Each version of Agones supports a specific version of Kubernetes. When a new version of Agones supports a new version of Kubernetes, it is explicitly called out in the [Agones version release notes](https://agones.dev/site/blog/releases/).
+
+The following table lists recent Agones versions and their corresponding required Kubernetes versions for Game Servers:
+
+| Agones version | Kubernetes version |
+| -------------- | ------------------ |
+| 1.27           | 1.23               |
```

Comment:
Should the top row pull the data from the `k8s-version` and `release-version` shortcodes?

Reply:
Good idea -- I'll add that in.

Reply:
I tried adding | {{% release-version %}} | {{% k8s-version %}} | as the top line, but when I ran `make site-server ENV=RELEASE_VERSION=1.20` the output was not as expected:

| Agones version | Kubernetes version |
| 1.20 | 1.23 |

Any idea if I'm using the shortcode incorrectly? Or is this an issue with the shortcode itself? Or an issue with the `make site-server` input args?

Reply:
The `k8s-version` shortcode is hardcoded ([here](https://github.com/googleforgames/agones/blob/main/site/layouts/shortcodes/k8s-version.html) and [here](https://github.com/googleforgames/agones/blob/main/site/config.toml#L94)), and there is no validation to ensure the mapping between it and the `release-version` is correct. I think to use `make site-server`, you have to use the correct  environment variable of `RELEASE_VERSION`, or just do not use any input args so it will pick the correct value [by default](https://github.com/googleforgames/agones/blob/main/site/layouts/shortcodes/release-version.html)

Reply:
If I'm understanding correctly, it would be better not to add in ` | {{% release-version %}} | {{% k8s-version %}} | ` because the `release-version` is variable and the  `k8s-version` is hard-coded?